arm-analysisservices/2016-05-16/swagger/analysisservices.json

```diff
@@ -410,6 +410,11 @@
       "description": "Properties of Analysis Services resource.",
       "type": "object",
       "properties": {
+        "state": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The current state of Analysis Services resource. It allows more custom terminal state like Paused."
```

Comment:
@ravbhatnagar
I suggest to add enums after service side make further change to differentiate provisioningState and state.
If you still want to have enums for now, I would add same value set for both of them.

Please let us know update.